packages/pdfx/lib/src/viewer/pinch/pdf_view_pinch.dart

```diff
@@ -385,7 +386,8 @@ class _PdfViewPinchState extends State<PdfViewPinch>
     final exposed = Rect.fromLTWH(
         -m.row0[3], -m.row1[3], _lastViewSize!.width, _lastViewSize!.height);
     final distBase = max(_lastViewSize!.height, _lastViewSize!.width);
-    for (final page in _pages) {
+    for (var i = 0; i < _pages.length; i++) {

```

Comment:
For what?

Reply:
When I open a pdf and close it quickly, it throws an error that this for is iterating a list that is empty, because when the page gets disposed, the _pages list gets cleared.